ruby/ql/test/library-tests/frameworks/action_view/CONSISTENCY/AstConsistency.expected

```diff
@@ -0,0 +1,2 @@
+extractionError
+| app/views/foo/bars/show.html.erb:10:8:11:7 | A parse error occurred. Check the syntax of the file. If the file is invalid, correct the error or exclude the file from analysis. |
```

Comment:
I think a `%>` is missing after the `do` .

Reply:
Even after adding `%>` there is still a parse error.

Reply:
Yeah the `key = [:display_text, :safe_text] do` phrase does not seem to make any sense. Perhaps they intended `for key in [:display_text, :safe_text] do` or similiar.

Reply:
Right. Fixed as well.